cookiecutter.json

```diff
@@ -1,6 +1,7 @@
 {
   "project_name": "my-mlops-project",
   "cloud": ["azure", "aws"],
+  "cicd_platform": ["azureDevOpsServices", "gitHub"],
```

Comment:
Why not have the existing selection (gitHub) first, as the default one? That way by default it preserves existing behavior.

Reply:
agreed. This is an artefact of what I was referencing in the above "outstanding issues" with "Currently, the cicd_platform will always be azureDevOpsServices as it is the first option listed in cookiecutter.json"

I'm still running into this issue and can't figure out (at least when I'm running the tests outlined in the readme) why the first option of the cicd_platform arg is always being selected. For example, currently if I have ["gitHub", "azureDevOpsServices"] in line 4, cookiecutter will always populate with "gitHub", regardless of the test config used. @vladimirk-db Would you be able to check if you run into the same issue when running those tests?

Reply:
Hmm not sure why the tests are not honoring selecting the non-default platform, can you post a link to the (failing/misbehaving) test run please? @smurching  would have more context. We should compare with the tests parameterizing "cloud" (azure/aws) since they would have had to overcome the same issue.

(My suggestion was to just swap the order in that cicd_platform options list, so that Github comes first to preserve existing default behavior.)

Reply:
the failure is resulting when I go to create the example projects as outlined in the [README](https://github.com/niall-turbitt/mlops-stack/blob/main/README.md#previewing-stack-changes). Regardless of the example config file specified, the resulting project will be whatever the first option of `cicd_provider` is.

For example, when the [cicd_platform options](https://github.com/niall-turbitt/mlops-stack/blob/48f04c36a7409f3504bf7568851de03aa2310351/cookiecutter.json#L4) are in the following order ["Azure DevOps", "GitHub Actions"] and I run the following:
`cookiecutter "$MLOPS_STACK_PATH" --config-file "$MLOPS_STACK_PATH/tests/example-project-configs/azure/azure-devops.yaml" --no-input --overwrite-if-exists`
the generated project will create the "Azure DevOps" version as expected. If the order of `cicd_platform` is reversed to ["GitHub Actions", "Azure DevOps"] the generated project will be the GitHub Actions version.

It does not honour the cicd choice in the example-project-config file specified, and will take the first option of the list for `cicd_platform` in `cookiecutter.json`. @smurching is this something you could validate?